frontend/src/employees/EmployeeListRow.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react'
+import { putUserInactive } from '../api/UserAPI';
+import EditButton from '../EditButton';
+import { Link } from 'react-router-dom';
+import Button from '../Button';
+
+export default function EmployeeListRow({user}) {
+
+    const [isActive, setIsActive] = useState(true)
```

Comment:
Ezt nem local state-ben kellene tárolni, hanem a user listát lehetne frissíteni, ha sikerült az inaktiválás.

Reply:
Gergővel beszéltünk multhéten ilyen optimistic módszerről, azt próbáltuk implementálni. Annyi problémába viszont tényleg ütköztünk, hogy a user lista csak component váltás után frissül, de akkor átírjuk.

Reply:
Igen, de az túl optimista, hogy nem kezeljük a hibát :) Ha nem sikerült a művelet, akkor utána vissza kell állítani az eredeti értékre.
@SiposGergo

Reply:
Szerintem a hiba kezeléséről is beszéltünk ennél a lehetőségnél :)